variables.tf

```diff
@@ -167,18 +161,18 @@ variable "ssm_tlscrypt_key" {
   default     = "/openvpn/server/tlscrypt.key"
 }
 
+variable "ssm_read_role_accounts_allowed" {
+  type        = list(string)
+  description = "List of accounts allowed to access the role that can read SSM keys."
```

Comment:
Just to keep this consistent with the other descriptions.

```suggestion
  description = "A list of accounts allowed to access the role that can read SSM keys."
```

Reply:
See commit d4302bc.